src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task implements Serializable {
```

Comment:
is it necessary to include `implements Serializable` here, when the parent class (Task) already implements it?

Reply:
same for the other classes that extend Task